packages/cardano-services/src/util/polling/DbSyncEpochPollService.ts

```diff
@@ -51,7 +51,7 @@ export class DbSyncEpochPollService implements EpochMonitor {
       name: 'current_epoch',
       text: findLastEpoch
     });
-    return Cardano.EpochNo(result.rows[0].no);
+    return Cardano.EpochNo(result.rowCount ? result.rows[0].no : 0);
```

Comment:
This looks like a separate fix, is it?

Reply:
Would be best to do in a separate commit then

Reply:
a62d450e8f should solve this